vector/src/main/java/im/vector/app/features/home/room/detail/timeline/item/AbsMessageItem.kt

```diff
@@ -75,7 +75,8 @@ abstract class AbsMessageItem<H : AbsMessageItem.Holder> : AbsBaseMessageItem<H>
             holder.timeView.visibility = View.VISIBLE
             holder.timeView.text = attributes.informationData.time
             holder.memberNameView.text = attributes.informationData.memberName
-            holder.memberNameView.setTextColor(attributes.getMemberNameColor())
+            // Tchap: Use primary color for text color. The color is set in the xml file.
+//            holder.memberNameView.setTextColor(ContextCompat.getColor(holder.view.context, R.color.palette_tchap_dark_blue))
```

Comment:
I would prefer changing the color in `MessageColorProvider.getMemberNameTextColor` and set a fixed color here.

Reply:
I don't have access to the context in 
```
MessageColorProvider.getMemberNameTextColor
```
if i would to return R.color.palette_tchap_dark_blue, i should use:
```
holder.memberNameView.setTextColor(ContextCompat.getColor(attributes.getMemberName())
```
I prefer the first option. What do you think ?

Reply:
You should not use dark_blue but a theme attribute, the dark blue will be... too dark in dark theme. I think you should use the `colorPrimary` or `colorSecondary` (which are the same in EA), or otherwise create a new tchap attribute for this. You can confirm this with @yostyle

Reply:
@yostyle prefers to use colorPrimary